BUILD.md

```diff
@@ -221,12 +231,13 @@ Required to support CPU and memory binding in the Legate launcher.
 Not available on conda; typically available through the system-level package
 manager.
 
-### MPI (`--openmpi` flag; optional)
+### MPI (optional)
 
 Only necessary if you wish to run on multiple nodes.
 
-Conda distributes a generic build of OpenMPI, but you may need to use a more
-specialized build, e.g. the one distributed by
+Environments created using the `--openmpi` flag of `generate-conda-envs.py` will
+contain the (generic) build of OpenMPI that is available on conda-forge. You may
+need/prefer to use a more specialized build, e.g. the one distributed by
 [MOFED](https://network.nvidia.com/products/infiniband-drivers/linux/mlnx_ofed/),
 or one provided by your HPC vendor. In that case you should use an environment
 file generated with `--no-openmpi`.
```

Comment:
So the `--openmp` flag when building legate is actually unrelated (does not matter?).

Reply:
Yeah, the `install.py` flag `--openmp` controls whether Legate will include support for OpenMP (no `I`). Very unfortunate name collision there.